tracing/src/main/java/org/axonframework/extensions/tracing/TracingQueryGateway.java

```diff
@@ -85,7 +80,7 @@ protected TracingQueryGateway(Builder builder) {
     @Override
     public <R, Q> CompletableFuture<R> query(String queryName, Q query, ResponseType<R> responseType) {
         Span parentSpan = tracer.activeSpan();
-        try (Scope scope = tracer.buildSpan(queryName).startActive(false)) {
+        try (Scope scope = tracer.buildSpan("send" + SpanUtils.messageName(query, queryName)).startActive(false)) {
```

Comment:
As mentioned in a comment on the `OpenTraceHandlerInterceptor`, I think we should add an underscore between `send` and the resolved message name.

Reply:
check